rainbow-me/rainbow
# Profiles: Support contenthash

Fixes TEAM2-434
Fixes TEAM2-440

## What changed (plus any additional context for devs)

This PR aims to fix a bug where the "Content" field in the create/edit profile flow was crashing. The reason for this is because we didn't polyfill `TextEncoder` (I think a merge conflict may have removed the polyfill we had earlier). 
It also turns out that setting a content hash never actually worked properly anyway. So this PR fixes that too.

## Screen recordings / screenshots

Adding a content hash:

https://www.loom.com/share/d705f44204304440a27f7b73ac2e3908

Removing a content hash:

https://www.loom.com/share/1997319b2e8948f5b0b1f01da28d8243


## What to test

- Ensure that you can successfully add a "Content" field by itself (setContenthash) on the Create/Edit Profile screen
- Ensure that you can successfully add a "Content" field with other fields (multicall) on the Create/Edit Profile screen
- Ensure that sending an ENS with the "Content" field populated, and the checkbox to clear records checked, that it clears the "Content" field.


## Final checklist
